packages/uui-tabs/lib/uui-tab-group.element.ts

```diff
@@ -26,13 +26,24 @@ export class UUITabGroupElement extends LitElement {
     `,
   ];
 
-  @queryAssignedNodes(undefined, true, 'uui-tab')
-  private slotNodes?: UUITabElement[];
+  @queryAssignedElements({
+    flatten: true,
+    selector: 'uui-tab, .uui-tab, [role=tab]',
+  })
+  private slotNodes?: HTMLElement[];
 
   private tabElements: UUITabElement[] = [];
 
   private setTabArray() {
-    this.tabElements = this.slotNodes ? this.slotNodes : [];
+    if (this.slotNodes) {
+      this.tabElements = this.slotNodes.filter(this.elementIsTab);
```

Comment:
Won't this remove anything else than the uui-tabs? Then you can't have <button role="tab">My Custom Tab</button> in the group?

Reply:
Won't exactly remove them, but it won't set an onclick listener on them and wont set the "active" property. I just thought we agreed on that your custom tabs must extend from uui-tab?

Reply:
I moved the check further down to when we actually invoke the "active" attribute